docs/source/delta-batch.md

```diff
@@ -742,6 +742,38 @@ Each time a checkpoint is written, Delta automatically cleans up log entries old
 .. note::
     Due to log entry cleanup, instances can arise where you cannot time travel to a version that is less than the retention interval. <Delta> requires all consecutive log entries since the previous checkpoint to time travel to a particular version. For example, with a table initially consisting of log entries for versions [0, 19] and a checkpoint at verison 10, if the log entry for version 0 is cleaned up, then you cannot time travel to versions [1, 9]. Increasing the table property `delta.logRetentionDuration` can help avoid these situations.
 
+### In-Commit Timestamps
+
+#### Overview
+<Delta> 3.3 introduced [In-Commit Timestamps](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#in-commit-timestamps) to provide a more reliable and consistent way to track table modifications. This feature addresses limitations of the traditional approach that relied on file modification timestamps, particularly in scenarios involving data migration or replication.
```

Comment:
Do we add delta-spark versions in our documentation?

Reply:
Looks like it has been done before. From the same file:

```
You can selectively overwrite only the data that matches an arbitrary expression. This feature is available with DataFrames in <Delta> 1.1.0 and above and supported in SQL in <Delta> 2.4.0 and above.
```